yarn.lock

```diff
@@ -698,17 +698,26 @@
     "@sentry/utils" "7.15.0"
     tslib "^1.9.3"
 
-"@sentry/hub@^7.13.0":
-  version "7.15.0"
-  resolved "https://registry.yarnpkg.com/@sentry/hub/-/hub-7.15.0.tgz#d4da91c15ab0a1ee2d70796bf7b2fda35f06837f"
-  integrity sha512-v15sSoYuKJ9+BmDUX6qxAnCDhlClmw6TY9/rcIYbP2XSxsGrJcPy6VPOw4E21/1zGXnKiW7KvBkPeYEjIx7fWA==
+"@sentry/core@7.17.3":
+  version "7.17.3"
+  resolved "https://registry.yarnpkg.com/@sentry/core/-/core-7.17.3.tgz#2b45c0507f1ef7018335b9bb61ed6b3f16accfad"
+  integrity sha512-PSboa9aOVnvZU+C6/shKlHUA7zjAl6z5BKRHF8mEljEYql6bh0HfJJKXtBHMz1sWnmzMa/qABSKLpnP5ZQlJNw==
   dependencies:
-    "@sentry/core" "7.15.0"
-    "@sentry/types" "7.15.0"
-    "@sentry/utils" "7.15.0"
+    "@sentry/types" "7.17.3"
+    "@sentry/utils" "7.17.3"
+    tslib "^1.9.3"
+
+"@sentry/hub@^7.16.0":
+  version "7.17.3"
+  resolved "https://registry.yarnpkg.com/@sentry/hub/-/hub-7.17.3.tgz#5c456d8523502957efd30e56e5b7c9c06e06306e"
+  integrity sha512-LxdA7q4Q8mLOHSubhtelURftZ7Ga8j4f3HY+f8yDEf/9fYr4XpEfVZknk1zIs2JhJPH0XCSvhyIIxT6zrudl3A==
+  dependencies:
+    "@sentry/core" "7.17.3"
+    "@sentry/types" "7.17.3"
+    "@sentry/utils" "7.17.3"
     tslib "^1.9.3"
 
-"@sentry/node@^7.13.0", "@sentry/node@^7.15.0":
+"@sentry/node@^7.15.0":
```

Comment:
looks like we're getting duplicate versions of things

Reply:
@scttcper not exactly sure why that is the case, mind if we look at this separately since it seems to have been there before?